app/decorators/models/solidus_address_name/address_decorator.rb

```diff
@@ -24,6 +24,22 @@ def self.value_attributes(base_attributes, merge_attributes = {})
       base.except(*DB_ONLY_ATTRS)
     end
 
+    def firstname
+      Spree::Address::Name.new(name).first_name
+    end
+
+    def lastname
+      Spree::Address::Name.new(name).last_name
+    end
+
+    def firstname=(name)
+      self.name = "#{name} #{lastname}"
+    end
+
+    def lastname=(name)
+      self.name = "#{firstname} #{name}"
+    end
```

Comment:
Does it mean it will save firstname and lastname to the `name`?
firstname and lastname will be empty?